### DISCUSSION TITLE
How to get GPU used?

### DISCUSSION THREAD

**Discussion by blueforest03** [microsoft/nni#5716] [Q&A]
I'm running the "Hello, NAS!" demo, and I did change the code in the "Launch an Experiment" section to get the algorithm to use GPU. But it just returned “Config is not provided. Will try to infer.” and "WARNING: GPU found but will not be used. Please set `experiment.config.trial_gpu_number` to the number of GPUs you want to use for each trial."
I'm using Python 3.11, PyTorch 2.0.1, CUDAToolkit: 11.7, cuDNN: 8.9, using jupyter via anaconda to run the code.
![(81)](https://github.com/microsoft/nni/assets/41872239/cdc1fd23-22fb-4a41-9568-d054f1ff4cb9)
Is there a method to use gou to run the code without bothering the shell or local file?(I want to use the jupyter to run the entire code)

**Comment by Baiyun-u-smartAI** [comment]
I got the same problem. And I failed to run in the end.
with the error like this:
[experiment.log](https://github.com/microsoft/nni/files/13627557/experiment.log)
